index.bs

```diff
@@ -6507,6 +6510,78 @@ when passed to a [=platform object=] expecting that type, and how IDL values
 of that type are <dfn id="dfn-convert-idl-to-ecmascript-value" export lt="converted to an ECMAScript value|converted to ECMAScript values">converted to ECMAScript values</dfn>
 when returned from a platform object.
 
+<div algorithm>
```

Comment:
Maybe. It's a bit more important here because of the grouping of abstract ops together with them, IMO, but it would probably be beneficial in both cases.